zcash_primitives/src/note_encryption.rs

```diff
@@ -20,6 +20,22 @@ use crate::{keys::OutgoingViewingKey, JUBJUB};
 pub const KDF_SAPLING_PERSONALIZATION: &'static [u8; 16] = b"Zcash_SaplingKDF";
 pub const PRF_OCK_PERSONALIZATION: &'static [u8; 16] = b"Zcash_Derive_ock";
 
+const COMPACT_NOTE_SIZE: usize = (
+    1  + // version
+    11 + // diversifier
+    8  + // value
+    32
+    // rcv
```

Comment:
Nit: It is not clear whether or not 32 and rcv go together. In the three previous values the comments are on the same line. It would be nice to be consistent, but I'm not too worried about it as I could see this being something the formatter did. *shrug*
(similar below)

Reply:
Yep, this is just how `rustfmt` wants to format it 🤷‍♂